server/fleetv2_http_api/impl/controllers.py

```
@@ -84,22 +84,25 @@ def login(device: Optional[str] = None) -> WerkzeugResponse | Response | tuple[d
     if device == "":
         try:
             auth_json = _security.device_get_authentication()
-        except:
+        except Exception as e:
             msg = "Problem reaching oAuth service."
-            return _log_and_respond(msg, 500, msg)
-        return _log_and_respond(auth_json, 200, "Device authentication initialized.")
+            _log_debug(str(e))
+            return _log_info_and_respond(msg, 500, msg)
+        return _log_info_and_respond(auth_json, 200, "Device authentication initialized.")
     elif device != None:
```

Ruff

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)